ext/tests/hooks_throw_exception_for_failed_call.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+Check if exceptions thrown in hooks interfere with internal exceptions
+--EXTENSIONS--
+opentelemetry
+--FILE--
+<?php
+function helloWorld($argument) {
+    vardump('inside');
```

Comment:
I think this is an intentional typo, could you annotate it so that somebody doesn't try to "fix" it in the future? Or break it more obviously.

Reply:
The typo was actually unintentional, but since the purpose was to show that this line is never reached (as the 'inside' is not in expected output), it didn't fail the test. Fixed it.